lib/api_endpoint/api_metamodel2spec.py

```diff
@@ -14,7 +14,7 @@ def get_path(
             enum_dict,
             operation_id,
             error_map,
-            enable_filtering):
```

Comment:
might as well make it global variable instead of passing it around to every single method. i know global variable generally is a bad idea but in this case i think it is okay to break the rule.

Reply:
yes, I tried remedying the usage of enable_filtering by adding it as a class  attribute to TypeHandler classes (also I renamed the parameter). I agree that rather than passing it everywhere, it is preferable to have a global. But substituting it with a global will affect almost all current test cases - I will delegate this change to a separate ticket.